src/color.js

```diff
@@ -286,21 +286,35 @@ exports.byElement = function() {
   }, null, null);
 };
 
-exports.bySS = function() {
+exports.bySS = function(grad) {
+  var palette;
+  if (grad) {
+    palette = {
+      C: grad._colors[0],
+      H: grad._colors[1],
+      E: grad._colors[2],
+    };
+  } else {
+    palette = {
+      C: [0.8, 0.8, 0.8, 1.0],
+      H: [0.6, 0.6, 0.9, 1.0],
+      E: [0.2, 0.8, 0.2, 1.0]
+    };
+  }
 
   return new ColorOp(function(atom, out, index) {
     switch (atom.residue().ss()) {
       case 'C':
-        out[index] = 0.8;   out[index+1] = 0.8; 
-        out[index+2] = 0.8; out[index+3] = 1.0;
+        out[index] = palette.C[0];   out[index+1] = palette.C[1]; 
```

Comment:
Sure, it's certainly also an option to accept both arrays and dictionary parameters.